x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/notes_tab.cy.ts

```diff
@@ -26,7 +26,12 @@ import { addNotesToTimeline, goToNotesTab } from '../../../tasks/timeline';
 
 import { deleteTimelines } from '../../../tasks/api_calls/common';
 
-const text = 'system_indices_superuser';
+const systemIndicesSuperuserName = 'system_indices_superuser';
+const serverlessCloudUserName = Cypress.env('ELASTICSEARCH_USERNAME');
```

Comment:
What is `ELASTICSEARCH_USERNAME` set to in ess? If it ends up being `system_indices_superuser` then you can just simplify all of this to be
`const author = Cypress.env('ELASTICSEARCH_USERNAME');` That's what it looks like for the [management section](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/management/cypress/cypress_base.config.ts#L52), but not sure about ours [here](https://github.com/elastic/kibana/blob/c5840b6a2b92201c51160bc53c4cb2b8f5ecf524/x-pack/test/security_solution_cypress/runner.ts#L22)

Reply:
You are completely right, changes pushed, thanks for the observation :)